lib/install/process.go

```diff
@@ -35,7 +35,6 @@ func InitProcess(ctx context.Context, installerConfig Config, gravityConfig proc
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	setLoggingOptions()
```

Comment:
I found it strange to be resetting logging configuration out of nowhere.
See below for changes for the similar effect.